src/acceptheaderpayee/handler.js

```diff
@@ -0,0 +1,449 @@
+      Logger.info((new Date().toISOString()), 'response: ', res.status)
+      if (res.status !== 202) {
+        // TODO: how does one identify the failed response?
+        throw new Error(`Failed to send. Result: ${res}`)
+      }
+
+      histTimerEnd({ success: true, fsp: 'payee', operation: 'postQuotes', source: req.headers['fspiop-source'], destination: req.headers['fspiop-destination'] })
+    } catch (err) {
+      Logger.error(err)
+      histTimerEnd({ success: false, fsp: 'payee', operation: 'postQuotes', source: req.headers['fspiop-source'], destination: req.headers['fspiop-destination'] })
+      // TODO: what if this fails? We need to log. What happens by default?
+      // const url = await rq.createErrorUrl(db, req.path, requesterName);
+      // TODO: review this error message
+      // TODO: we should be able to throw an AppError somewhere, test whether the error
+      // received in this handler is an AppError, then send the requester the correct
+      // payload etc. based on the contents of that AppError.
+      // rq.sendError(url, asyncResponses.serverError, rq.defaultHeaders(requesterName, 'participants'), {logger});
+    }
+  })()
+  return h.response().code(202)
+}
+
+exports.postTransfers = async function (req, h) {
+  const histTimerEnd = Metrics.getHistogram(
+    'sim_request',
+    'Histogram for Simulator http operations',
+    ['success', 'fsp', 'operation', 'source', 'destination']
+  ).startTimer()
+
+  Logger.debug(`[cid=${req.payload.transferId}, fsp=${req.headers['fspiop-source']}, source=${req.headers['fspiop-source']}, dest=${req.headers['fspiop-destination']}] ~ Simulator::api::payee::postTransfers - START`)
+
+  const metadata = `${req.method} ${req.path} ${req.payload.transferId}`
+  Logger.info(`IN PAYEEFSP:: received: ${metadata}.`)
+
+  if (!transfersFulfilResponseDisabled) {
+    // Saving Incoming request
+    const incomingRequest = {
+      headers: req.headers,
+      data: req.payload
+    }
+    requests.set(req.payload.transferId, incomingRequest)
+
+    const url = transfersEndpoint + '/transfers/' + req.payload.transferId
+    const fspiopUriHeader = `/transfers/${req.payload.transferId}`
+    try {
+      const transfersResponse = {
+        fulfilment: transfersFulfilment,
+        completedTimestamp: new Date().toISOString(),
+        transferState: 'COMMITTED'
+      }
+      const protectedHeader = {
+        alg: 'RS256',
+        'FSPIOP-Source': `${req.headers['fspiop-destination']}`,
+        'FSPIOP-Destination': `${req.headers['fspiop-source']}`,
+        'FSPIOP-URI': `/transfers/${req.payload.transferId}`,
+        'FSPIOP-HTTP-Method': 'PUT',
+        Date: ''
+      }
+      const fspiopSignature = {
+        signature: signature,
+        protectedHeader: `${base64url.encode(JSON.stringify(protectedHeader))}`
+      }
+      const opts = {
+        method: 'PUT',
+        headers: {
+          'Content-Type': 'application/vnd.interoperability.transfers+json;version=1.0',
```

Comment:
why is the Accept header not here? i thought that this was to test if the accept header is sent on a PUT

Reply:
The `Accept` header is set by default.